pixeltable/exprs/arithmetic_expr.py

```diff
@@ -25,17 +25,34 @@ def __init__(self, operator: ArithmeticOperator, op1: Expr, op2: Expr):
         if op1.col_type.is_json_type() or op2.col_type.is_json_type() or operator == ArithmeticOperator.DIV:
             # we assume it's a float
             super().__init__(ts.FloatType(nullable=(op1.col_type.nullable or op2.col_type.nullable)))
+        elif op1.col_type.is_string_type() and operator in (ArithmeticOperator.ADD, ArithmeticOperator.MUL):
```

Comment:
Let's create a new expr subtype for string operations. This isn't really arithmetic, and it'll be easier to special-case later.

Reply:
yup perhaps better use case is to support string formatting f strings,  str.format etc

Reply:
Alternatively, we could rename `ArithmeticExpr` to something more general, such as `OperatorExpr`.